### DISCUSSION TITLE
Not full width

### DISCUSSION THREAD

**Discussion by Satont** [community/community#73166] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I'm not found similar discussions, my feed is not taking full width of my screen, i need to set github zoom to 125, otherwise it's looks like that:

![image](https://github.com/community/community/assets/42675886/7824e3b3-2ea2-4ae0-9593-cb460d6957fb)

**Comment by Esra404** [comment]
A tool which helps in demonstrations, presentations or if you just need to explain someone something on your computer. DemoHelper is an annotation and screen zoom tool you can use for technical presentations that include application demonstrations.
**[Tools.stefankueng.com/DemoHelper.html](https://tools.stefankueng.com/DemoHelper.html)**

**Comment by ALNPRA123** [comment]
you can make width inside css : setting in 100%,after that you can take this syntax about display : block,same device or machine software not fully support  for code editor ,i recommended you can use about browser chrome,because google have that and i think big company start up just a google have a best software,hope can i help you bro

**Comment by BrijeshLathiya** [comment]
In most web browsers, including Chrome, Firefox, Safari, and Edge, you can perform a "hard refresh" using the following keyboard shortcuts:

Windows/Linux:

Google Chrome, Mozilla Firefox, Microsoft Edge: Ctrl + F5
Safari: Shift + Ctrl + R

Mac:
Google Chrome, Mozilla Firefox, Microsoft Edge: Cmd + Shift + R
Safari: Cmd + R
